code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm

```diff
@@ -136,6 +136,13 @@
 	sort_category = "Xenowear - Skrell"
 	flags = GEAR_NO_SELECTION
 
+/datum/gear/wulu_pouch
+	display_name = "wulumunusha joint pouch"
+	path = /obj/item/storage/box/fancy/cigarettes/wulu
+	whitelisted = list(SPECIES_SKRELL, SPECIES_SKRELL_AXIORI)
```

Comment:
why? can't anyone smoke wulumunusha and/or grow it?

Reply:
These are from the Lyukal. A Skrell resistance movement. Normal humans wouldn’t necessarily have the connections to get them from the Lyukal. Plus, it has a significant religious meaning for Skrell where-as any other species would just use them for drugs.

Reply:
other changes are addressed, gonna just let this one sit until a decision is made. 

probably comes down to whether there wants to be a risk of anyone having access to doing lrp  stoner stuff via loadout, because other loadout items are restricted that are no more difficult to get access to (origin-specific drip, probably orderable from space amazon realistically, but just restricted for ease of not having to police it)

i have no take either way since i'm just the guy putting the pr up, but that was roughly the logic in my head whilst i was doing this. i wasn't sure if drugs in loadout would be allowed even for specifically just skrell, frankly

Reply:
fine by me.